cfme/tests/services/test_rest_services.py

```diff
@@ -449,3 +455,73 @@ def test_edit_blueprints(self, rest_api, blueprints, from_detail):
         assert len(edited) == response_len
         for i in range(response_len):
             assert edited[i].ui_properties == new[i]['ui_properties']
+
+
+class TestOrchestrationTemplatesRESTAPI(object):
+    @pytest.mark.tier(3)
+    @pytest.mark.uncollectif(lambda: version.current_version() < '5.7')
+    def test_create_orchestration_templates(self, rest_api, orchestration_templates):
+        """Tests creation of orchestration templates.
+
+        Metadata:
+            test_flag: rest
+        """
+        assert len(orchestration_templates) > 0
+        record = rest_api.collections.orchestration_templates.get(id=orchestration_templates[0].id)
```

Comment:
I think it's worth checking them all, if there was an error where only the first was processed, and we've all seen issues like that before......